### ISSUE TITLE
IntelliJ Plugin format (Ctrl+Alt+L) diverges from command line (google-java-format.jar)

### ISSUE THREAD

**Issue by habutre** [google/google-java-format#919]
Expected behavior:

Not much changes beyond described in the release notes

Actual behavior:

Indentation changes dramatically

Before:

```
      @RequestParam(value = "createdAt") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @NotNull
          LocalDate createdAt) {
    
```

After:

```
      @RequestParam(value = "createdAt") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @NotNull
      LocalDate createdAt) {   
```

I have a pre-commit hook script that prevents unformatted to be committed and now the formats diverges between IntelliJ Plugin (`Ctrl+Alt+L`) and the command `google-java-format.jar --skip-reflowing-long-strings`

*Environment details:*

```
IntelliJ IDEA 2022.3.3 (Community Edition)
Build #IC-223.8836.41, built on March 10, 2023
Runtime version: 17.0.6+1-b653.34 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.19.17-2-manjaro
GC: G1 Young Generation, G1 Old Generation
Memory: 1536M
Cores: 8
Registry:
    debugger.watches.in.variables=false
    debugger.new.tool.window.layout=true
    ide.experimental.ui=true

Non-Bundled Plugins:
    google-java-format (1.16.0.0)
    gherkin (223.8617.8)
    cucumber-java (223.7571.123)
    org.sonarlint.idea (8.0.0.63273)

Kotlin: 223-1.8.0-release-345-IJ8836.41
Current Desktop: GNOME
```

**Comment by plumpy** [comment]
I can't reproduce this. Are you sure the plugin is enabled and [configured correctly](https://github.com/google/google-java-format/blob/master/README.md#intellij-jre-config)?

A quick way to check is by creating a method like

```
void foo() {
}
```

And seeing if it gets reformatted onto a single line. (The built-in formatter generally won't do that. `google-java-format` definitely will.)

**Comment by plumpy** [comment]
I suspect what's happening is that your IDE doesn't have the proper `--add-exports` lines. It will display a notification when you open the project, but that's easy to miss. And then it will fall back to using the the built-in IntelliJ formatter.

I'm going to publish a new version of the plugin to fix this behavior. It will show the notification every time you try to run the formatter and will no longer fall back to the built-in formatter.

If that doesn't seem like your issue feel free to reopen this.

**Comment by plumpy** [comment]
Fixed in 1.16.0.1.

**Comment by habutre** [comment]
For some unexplainable reason (of course my bad) the following line was missing: :man_shrugging: 

```
--add-exports=jdk.compiler/com.sun.tools.javac.code=ALL-UNNAMED
```

Thanks for the speed light fix and caring :trophy:
